Examples/test-suite/csharp_argument_defaults_feature.i

```diff
@@ -0,0 +1,40 @@
+%module csharp_argument_defaults_feature
+%include "std_string.i"
+
+
+%feature("cs:defaultargs") Foo::Foo;
+%feature("cs:defaultargs", z=4) Foo::bar;
+%feature("cs:defaultargs", x="\"fives\"") Foo::zoo;
+%feature("cs:defaultargs", value="System.Math.PI") Foo::pi;
```

Comment:
A test demonstrating C++11 raw strings would be useful here.

Reply:
There's one just below (possibly added after your comment?)

It'd be helpful to mention you can use C++11 raw strings in this context in the manual too though.